great_expectations/expectations/core/expect_column_kl_divergence_to_be_less_than.py

```diff
@@ -327,9 +327,13 @@ class ExpectColumnKLDivergenceToBeLessThan(ColumnAggregateExpectation):
     tail_weight_holdout: Union[float, None] = pydantic.Field(
         default=0, ge=0, le=1, description=TAIL_WEIGHT_HOLDOUT_DESCRIPTION
     )
-    bucketize_data: bool = pydantic.Field(True, description=BUCKETIZE_DATA_DESCRIPTION)
-    min_value: Optional[Comparable] = pydantic.Field(None, description=MIN_VALUE_DESCRIPTION)
-    max_value: Optional[Comparable] = pydantic.Field(None, description=MAX_VALUE_DESCRIPTION)
+    bucketize_data: bool = pydantic.Field(default=True, description=BUCKETIZE_DATA_DESCRIPTION)
+    min_value: Optional[Comparable] = pydantic.Field(
+        default=None, description=MIN_VALUE_DESCRIPTION
```

Comment:
i'm curious, why all the updates to the kwarg `default=None`? i appreciate it for readability, but wondering if there's a functional reason.

Reply:
Responded in slack too with the longer answer, but it's to make mypy understand